### DISCUSSION TITLE
The future of `now.sh` subdomains?

### DISCUSSION THREAD

**Discussion by ahmed2m** [vercel/vercel#5969] [General]
Today I saw an error: New `now.sh` are deprecated, what does this mean for existing projects with a now.sh domain and will there be any problems for them such as no future preview deploys or a force redirect to a `vercel.app` subdomain? 

Also was that decision discussed publicly? And we won't ever have the ability to create new `now.sh` domains not even in the distant future? 

![image](https://user-images.githubusercontent.com/26524089/110843366-de474600-82b0-11eb-9497-ef47828d0e6a.png)

**Comment by fu-sen** [comment]
The Vercel Team should have announced by email in January. You overlooked:
https://vercel.com/changelog/urls-are-becoming-consistent

Anyway, `now.sh` is no longer available and they have been unified into` vercel.app`.
